src/main/java/racingcar/domain/Cars.java

```diff
@@ -0,0 +1,31 @@
+package racingcar.domain;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Cars {
+
+    private final List<Car> cars;
+
+    public Cars(final List<Car> cars) {
+        this.cars = cars;
+    }
+
+    public List<Car> filterWinners() {
+        int max = calculateMax();
+        return cars.stream()
+            .filter(car -> car.distance() == max)
+            .collect(Collectors.toList());
+    }
+
+    public List<Car> get() {
+        return cars;
+    }
+
+    private int calculateMax() {
+        return cars.stream()
```

Comment:
`mapToInt`를 활용하면 아래와 같이 작성 가능해요!

```suggestion
        return cars.stream()
            .mapToInt(Car::distance)
            .max()
            .orElseThrow(IllegalArgumentException::new)
```

Reply:
Exception을 발생시킬때는 에러 메세지를 넣어주면 이슈 파악하기가 훨씬 쉬울 것 같아요~